ledger-core-bitcoin/idl/addresses.djinni

```diff
@@ -79,3 +79,13 @@ BitcoinLikeExtendedPublicKey = interface +c {
     # Get the root path of the xPUB.
     getRootPath(): string;
 }
+
+# A keychain can be seen as an address database that can be queried to get lists of addresses,
+# ask whether an address is contained, or generate new addresses.
+Keychain = interface +c {
```

Comment:
On the legacy, I have added that interface in the “common” part of the library, [here](https://github.com/LedgerHQ/lib-ledger-core/blob/develop/core/idl/wallet/common/wallet.djinni#L317-L323). So I think this should belong to the `ledger-core` project, not necessarily to `ledger-core-bitcoin`.

Reply:
It was the recommendation of Pierre to add it to the bitcoin interface only, may be because it does not make sense when we have only one adresse to return with the other currencies.